README.md

```diff
@@ -1,6 +1,16 @@
-# CyBear Jinni Smart Device
+<h1 align="center">CyBear Jinni Smart Device</h1>
 
-Welcome
+![CyBear Jinni](https://user-images.githubusercontent.com/9304740/94843279-24a49900-0425-11eb-83f3-87e8ba40b1dc.png)
+
+![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)
+
+[![style: effective dart](https://img.shields.io/badge/style-effective_dart-40c4ff.svg)](https://pub.dev/packages/effective_dart)
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
```

Comment:
All the badges should be in the same line

Reply:
Open source badge should not link to the github of the badge.
You can link it to the [wikipedia](https://en.wikipedia.org/wiki/Open_source) page of it or get creative.

Reply:
sure